package.json

```diff
@@ -38,5 +38,13 @@
       "web/packages/teleterm",
       "e/web/**"
     ]
+  },
+  "dependencies": {
```

Comment:
This doesn't seem right. These updates should probably go in the nested package.json files.

Reply:
Thank you for mentioning this.  I was a little confused by this also.

This was committed as part of the `yarn upgrade` command, but I noticed the dependabot PR's did not have to do this.  I was hoping someone could explain to me a reliable way to make sure the dependency is updated without making the `package.json` change.

What is the correct way to do this?

Reply:
Thank you @hatched, I removed these changes and applied the `make clean-ui`.  Let me know how this looks now.